src/ipyvizzu/data/converters/spark/converter.py

```diff
@@ -48,8 +48,13 @@ def __init__(
         default_measure_value: MeasureValue = NAN_MEASURE,
         default_dimension_value: DimensionValue = NAN_DIMENSION,
         max_rows: int = MAX_ROWS,
+        units: Optional[Dict[str, str]] = None,
     ) -> None:
-        super().__init__(default_measure_value, default_dimension_value, max_rows)
+        # pylint: disable=too-many-arguments
```

Comment:
Can you separate these constructor arguments (defaults, max_rows, units(?)) into a "settings" class?